web/migrations/0050_badge_custom_type_alter_badge_badge_type_and_more.py

```
@@ -0,0 +1,47 @@
+            ),
+        ),
+        migrations.AlterField(
+            model_name="badge",
+            name="description",
+            field=models.TextField(blank=True),
+        ),
+        migrations.AlterField(
+            model_name="badge",
+            name="image",
+            field=models.ImageField(
+                blank=True, help_text="Upload an image for the badge (200x200 recommended)", upload_to="badge_images/"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)